docs/src/components/MainSectionCard.js

```diff
@@ -9,18 +9,20 @@ import theme from './atomDark.js';
 import Markdown from './Markdown.js';
 
 type Props = {|
-  cardSize?: 'sm' | 'md',
-  defaultCode: string,
+  cardSize?: 'sm' | 'md' | 'lg',
+  children?: Node,
```

Comment:
ditto here

Reply:
Same here, sometimes you can render easily with defaultCode, other times you need to pass in children (CombinationNew)